cico_functions.sh

```diff
@@ -86,13 +113,16 @@ function build_and_push() {
 
   # Let's build and push image to 'quay.io' using git commit hash as tag first
   set_git_commit_tag
-  docker build -t ${IMAGE} -f ./build/dockerfiles/${DOCKERFILE} . | cat
-  tag_push "${REGISTRY}/${ORGANIZATION}/${IMAGE}:${GIT_COMMIT_TAG}"
-  echo "CICO: '${GIT_COMMIT_TAG}' version of images pushed to '${REGISTRY}/${ORGANIZATION}' organization"
 
-  # If additional tag is set (e.g. "nightly"), let's tag the image accordingly and also push to 'quay.io'
+  # Create a new builder instance using buildx 
+  docker buildx create --use --name builder
+  docker buildx inspect --bootstrap
+  docker buildx build --platform linux/amd64,linux/s390x -t ${REGISTRY}/${ORGANIZATION}/${IMAGE}:${GIT_COMMIT_TAG} -f ./build/dockerfiles/${DOCKERFILE} --push --progress plain --no-cache .
+  echo "CICO: '${GIT_COMMIT_TAG}' version of images pushed to '${REGISTRY}/${ORGANIZATION}' organization"
+  
```

Comment:
Can we extract it to a function similar to the removed `tag_push()`

Reply:
Extracted `docker buildx` command to a separate function (`build_and_push_using_buildx`).